adafruit_magtag/network.py

```diff
@@ -139,7 +139,7 @@ def json_traverse(json, path):
         for x in path:
             try:
                 value = value[x]
-            except TypeError as error:
+            except (TypeError, KeyError) as error:
```

Comment:
Do you want IndexError as well? I think that would cover lists. KeyError is for dicts I believe.

Reply:
Thanks, I'll add that too.